jobs/ci-kubernetes-e2e-kubeadm-gce-1-7.env

```diff
@@ -0,0 +1,13 @@
+### job-env
+
+PROJECT=k8s-jkns-gci-gke-reboot-1-4
```

Comment:
yes, we are recycling old projects. We have probably 200+ inactive ones and also looking into better projects management.